libs/clients/taktikal/src/lib/taktikal.service.spec.ts

```diff
@@ -0,0 +1,100 @@
+      pdfFileName: 'example.pdf',
+      flowKey: '7e364abfaxx5', // available at https://app-dev.taktikal.is/settings/access-keys
+      meta: { myKey: 'myvalue' },
+      user: 'your@email.is',
+      signInOrder: false,
+      requiresAuth: false,
+      signatureLocation: CreateSigningProcessSignatureLocationEnum.TopFirstPage,
+      createSignees: [
+        {
+          name: 'Test User',
+          ssn: '1234567890',
+          address: 'Address 5',
+          postalCode: '555',
+          city: 'Reykjavík',
+          email: 'testUser@taktikal.is',
+          phoneNumber: '1234567',
+          communicationDeliveryType: 'none', // can be "none, email, sms" if the SignatureType is Simple, SMS can not be used
+          signatureType: 'Qualified', // Can be Qualified or Simple
+          hidePersonalCode: false, // ssn will be hidden in the signature appearance if true
+          reason: 'Undirritun',
+          language: 'En', // optional parameter
```

Comment:
The comments provided here for options should rather be in examples provided in the README